draft-nuttycom-funding-allocation.rst

```diff
@@ -0,0 +1,609 @@
+::
+
+  ZIP: Unassigned
+  Title: Block Reward Allocation for Non-Direct Development Funding
+  Owners: Kris Nuttycombe <kris@nutty.land>
+          Jason McGee <aquietinvestor@gmail.com>
+  Original-Authors: Skylar Saveland <skylar@free2z.com>
+  Credits: Daira-Emma Hopwood
+           Jack Grigg
+           @Peacemonger (Zcash Forum)
+  Status: Draft
+  Category: Consensus
+  Created: 2024-07-03
+  License: MIT
+  Pull-Request: <https://github.com/zcash/zips/pull/866>
+
+Terminology
+===========
+
+The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this
+document are to be interpreted as described in BCP 14 [#BCP14]_ when, and only
+when, they appear in all capitals.
+
+Abstract
+========
+
+This ZIP proposes several options for the allocation of a percentage of the
+Zcash block subsidy, post-November 2024 halving, to an in-protocol "lockbox."
+The "lockbox" will be a separate pool of issued funds tracked by the protocol,
+as described in ZIP <TBD>: Lockbox Funding Streams
+[#zip-lockbox-funding-streams]_. No disbursement mechanism is currently defined
```

Comment:
These reference names get changed to be the actual ZIP number once that is assigned. While draft, they should match the draft name:
```suggestion
[#draft-nuttycom-lockbox-streams]_. No disbursement mechanism is currently defined
```